### PR TITLE
Adding vuid 03572, Validate acceleration structure bindings count

### PR DESCRIPTION
**Pull request by ziga-lunarg**

### CODE REVIEW

File: tests/vklayertests_descriptor_renderpass_framebuffer.cpp [hunk 1]
```diff
@@ -10287,3 +10287,58 @@ TEST_F(VkLayerTest, DescriptorUpdateOfMultipleBindingWithOneUpdateCall) {
...
+    for (uint32_t i = 0; i <= maxBlocks; ++i) {
+        dslb.binding = i;
+        dslb_vec.push_back(dslb);
+    }
+
+    VkDescriptorSetLayoutCreateInfo ds_layout_ci = LvlInitStruct<VkDescriptorSetLayoutCreateInfo>();
+    ds_layout_ci.bindingCount = dslb_vec.size();
+    ds_layout_ci.pBindings = dslb_vec.data();
+
+    VkDescriptorSetLayout ds_layout;
```
Comment (jeremyg-lunarg) [inline tests/vklayertests_descriptor_renderpass_framebuffer.cpp:10333]: Looks like ds_layout will be leaked. vk_testing::DescriptorSetLayout

**Review by jeremyg-lunarg — CHANGES_REQUESTED** [review]
Probably a leak in the test case, otherwise looks good.

File: tests/vklayertests_descriptor_renderpass_framebuffer.cpp (hunk 1 above)
Comment (ziga-lunarg) [reply tests/vklayertests_descriptor_renderpass_framebuffer.cpp:10333]: Will do, thanks

**Review by jeremyg-lunarg — APPROVED** [review]
LGTM

**Pull request merged**